### PR TITLE
WIP: Adding tomopy separately from its deps.

### PR DESCRIPTION
**Pull request by ericdill**
See #681 for tomopy deps

### CODE REVIEW

File: recipes/tomopy/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+        - pyfftw
+        - python
+        - pywavelets
+        - numpy
+        - scikit-image
+        - scipy
+        - six
+
+test:
+    # Python imports
```
Comment (ocefpaf) [inline recipes/tomopy/meta.yaml]: @ericdill Can you remove this comment?

PS:  is this recipe good to go?

Comment (ericdill) [inline recipes/tomopy/meta.yaml]: Done.

**Comment by ericdill** [comment]
@ocefpaf yes this recipe is good to go. Thanks for the review!

File: recipes/tomopy/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+    version: {{ version }}
+
+source:
+    url: https://github.com/tomopy/tomopy/archive/{{ version }}.tar.gz
+    fn: tomopy-{{ version }}.tar.gz
+    sha256: b30a4828c9928aacb26f3cd1f44555aeeb031e9ed3585014b04a3b5079e78e24
+
+build:
+    number: 0
+    script: C_INCLUDE_PATH=$PREFIX/include python setup.py install --single-version-externally-managed --record=record.txt
```
Comment (ocefpaf) [inline recipes/tomopy/meta.yaml]: `$PREFIX/include` will break if you have Windows builds one day.

**Pull request merged**